### PR TITLE
chore: add v1v and ifernandezcalvo as developers to OpenTelemetry plugin

### PR DESCRIPTION
**Pull request by kuisathaverat**
# Description

This PR adds @v1v and @kuisathaverat (ifernandezcalvo) as developers in the list 

https://github.com/jenkinsci/opentelemetry-plugin

@cyrille-leclerc

# Submitter checklist for adding or changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [ ] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@daniel-beck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by NotMyFault** [comment]
@v1v You need to log into https://issues.jenkins.io/ and https://repo.jenkins-ci.org/ in order to get a green check here.

**Review by cyrille-leclerc — APPROVED** [review]

**Comment by NotMyFault** [comment]
@v1v To move on with this PR, please log in to artifactory and jira.

**Comment by v1v** [comment]
> @v1v To move on with this PR, please log in to artifactory and jira.

Need to find out those details, somehow I have a few users, so I'll do it later on and keep you posted when it's done, Thanks for the reminder 🙇

**Comment by v1v** [comment]
I've just accessed both systems (Jira and Artifactory), I guess I might need to wait for a couple of hours until the validation can be retriggered

**Comment by NotMyFault** [comment]
No worries and thanks for doing both :)

Is the name you used to log in on both services `v1v`? I'm asking because neither artifactory nor jira have an account with that name: https://issues.jenkins.io/secure/ViewProfile.jspa?name=v1v

**Comment by v1v** [comment]
> s the name you used to log in on both services v1v? I'm asking because neither artifactory nor jira have an account with that name: https://issues.jenkins.io/secure/ViewProfile.jspa?name=v1v

It's a bit of a mess on my end. `v2v` is the one in Jenkins.

File: permissions/plugin-opentelemetry.yml [hunk 1]
```diff
@@ -5,5 +5,7 @@ paths:
   - "io/jenkins/plugins/opentelemetry"
 developers:
   - "cleclerc"
+  - "ifernandezcalvo"
+  - "v1v"
```
Comment (v1v) [inline permissions/plugin-opentelemetry.yml:9]:
```suggestion
  - "v2v"
```

**Comment by NotMyFault** [comment]
No worries, thanks for correcting the username!

**Review by NotMyFault — APPROVED** [review]

**Pull request merged**